src/embed/liveboard.ts

```diff
@@ -338,6 +338,45 @@ export interface LiveboardViewConfig
      * ```
      */
     hideIrrelevantChipsInLiveboardTabs?: boolean;
+
+    /**
+     * The Liveboard to run on regular intervals to fetch the cdw token.
+     * @hidden
+     * @version SDK: 1.35.0 | ThoughtSpot:10.6.0.cl
+     * @example
+     * ```js
+     * const embed = new LiveboardEmbed('#embed-container', {
+     *    ... // other options
+     *    oAuthPollingInterval: value in milli seconds,
```

Comment:
value in millisecond as string or '300000'

Reply:
value is in milliseconds as number like  '300000'